lib/tapioca/commands.rb

```diff
@@ -0,0 +1,9 @@
+# typed: true
+# frozen_string_literal: true
+
+require_relative "commands/command"
```

Comment:
Would it make sense to autoload these classes? We probably don't need to have all command classes required at all times, since we can only run one at a time.

Reply:
Done